lib/controllers/ServiceFabrikAdminController.js

```diff
@@ -290,6 +290,7 @@ class ServiceFabrikAdminController extends BaseController {
       .assign(req.body)
       .set('user', req.user)
       .value();
+    options.tenant_guid = req.body.space_guid;
```

Comment:
options would still have `space_guid` as its creation involves assigning attribs from `req.body` . Could be a good idea to change Line 290 to `.assign(_.omit(req.body, 'space_guid'))`. Doing so ensures that any downstream code relying on this will fail, else it might go unnoticed.

Reply:
Agree...will make this change.